pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -28,10 +28,16 @@ type ofTable struct {
 }
 
 func (t *ofTable) GetID() TableIDType {
+	t.RLock()
+	defer t.RUnlock()
```

Comment:
Could `t.id` be read/write concurrently?

Reply:
Thanks for pointing out. Then for ofTable,  we only need to use Lock for UpdateStatus() and RLock for Status(). Will update this part.

Reply:
t.ID is added when create the ofTable, and is only readable at other moments.

Reply:
OK, got it.